docs/core-concepts/api-calls-anatomy/index.md

```diff
@@ -47,62 +57,113 @@ Every request data has several required fields which you must provide them and t
 Request data for this call is like so:
 
 ```ts
-interface IAssetIndex {
-  residence_list: 1; // it must always be `1`
-  passthrough?: object;
-  req_id?: number;
+{
+  residence_list: 1; // Api Call Method Name
+  passthrough?: object; // Optional
+  req_id?: number; // Optional
 }
 ```
 
-The `residence_list` field is the `Identifier Field` for the call and is required here, there may other required fields which are related to type of the request you wanna send. if you want to know more about `Residence List` check [it](https://api.deriv.com/api-explorer#residence_list) out here.
+The `residence_list` field is the `method name` for the call and is required, there may be other required fields which are related to type of the request you wanna send. if you want to know more about `Residence List` and other API calls please check them out in [API Explorer](https://api.deriv.com/api-explorer#residence_list).
```

Comment:
Text adjustment:

The `residence_list` field is the `method name` for the call and is required. There may be other required fields which are related to this type of the request you want to send. if you want to know more about `Residence List` and other API calls please check them out in the [API Explorer](https://api.deriv.com/api-explorer#residence_list).

Reply:
done